core/src/Cabana_Parallel.hpp

```diff
@@ -251,7 +253,8 @@ template <class FunctorType, class NeighborListType, class... ExecParameters>
 inline void neighbor_parallel_for(
     const Kokkos::RangePolicy<ExecParameters...>& exec_policy,
     const FunctorType& functor, const NeighborListType& list,
-    const FirstNeighborsTag, const SerialOpTag, const std::string& str = "" )
+    const FirstNeighborsTag, const SerialOpTag, const std::string& str = "",
+    typename std::enable_if<( !is_linked_cell_list<NeighborListType>::value ), int>::type* = 0 )
```

Comment:
Currently checking that it is not a linked cell list for non-linked cell neighbor parallel as I didn't see a `is_neighbor_list` check. Not sure if this is the best way to differentiate the linked cell approach from this approach.

Reply:
Yeah, we don't have one. If we figure out the `getNeighbor` we can remove it anyway (but it would require some kind of other tag for the LCL only variant)